### PR TITLE
Host uxa

### PR DESCRIPTION
**Pull request by deetz99**
*Issue:*

- bcgov/entity/issues/25001

*Description of changes:*
- several fixes according to figma uxa file
- done everything except for items with question marks, number of rooms scrolling stuff and making residential postal code not required
- update strata dashboard column date format and column width to match same columns in hosts

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: strr-base-web/app/app.config.ts
Comment (deetz99) [inline strr-base-web/app/app.config.ts:1]: radio group gap 16px

File: strr-base-web/app/components/connect/fee/Widget.vue
Comment (deetz99) [inline strr-base-web/app/components/connect/fee/Widget.vue:1]: remove extra border at bottom of widget

File: strr-base-web/app/components/connect/form/field/Group.vue
Comment (deetz99) [inline strr-base-web/app/components/connect/form/field/Group.vue:1]: min/max attrs for number inputs

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-408-gggmew06.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-hosts-dev--pr-408-w9gl20mc.web.app

File: strr-host-pm-web/app/components/form/AddOwners/input/Business.vue
Comment (deetz99) [inline strr-host-pm-web/app/components/form/AddOwners/input/Business.vue:1]: add missing location description v-model

File: strr-host-pm-web/app/components/form/AddOwners/input/Person.vue
Comment (deetz99) [inline strr-host-pm-web/app/components/form/AddOwners/input/Person.vue:1]: add missing location description v-model

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-strata-hotel-dev--pr-408-gggmew06.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-hosts-dev--pr-408-w9gl20mc.web.app

File: strr-host-pm-web/app/components/form/DefineYourRental/UnitRequirements/List.vue
Comment (deetz99) [inline strr-host-pm-web/app/components/form/DefineYourRental/UnitRequirements/List.vue:1]: add dividers to requirements list

**Review by dimak1 — APPROVED** [review]
LGTM!

**Pull request merged**